lib/src/helper.dart

```diff
@@ -179,4 +179,16 @@ class Helper {
       throw Exception('requestCapturePermission only support for Android');
     }
   }
+
+  static Future<bool> configureIOSMultitaskingCameraAccess(bool enable) async {
+    if (WebRTC.platformIsIOS) {
+      return await WebRTC.invokeMethod(
+        'configureIOSMultitaskingCameraAccess',
+        <String, dynamic>{'enable': enable},
+      );
+    } else {
+      throw Exception(
+          'configureIOSMultitaskingCameraAccess is only supported for iOS');
+    }
+  }
```

Comment:
IMO `configureIOSMultitaskingCameraAccess(true)` doesn't really make a ton of sense to a developer. Can we either change the enable to a required named parameter or maybe change this to enableIOSMultitaskingCamerAccess? If there's a good reason to keep it like this, I can approve.